Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM redwolfgang20/grails as builder
+COPY . /app
+WORKDIR /app
+
+RUN ./gradlew webpackProd
+RUN grails -Dgrails.env=dev war
```

Comment:
This doesn't seem right ... env=dev?

Reply:
Following up on this. We do need to know how the image should be run, and that might be done as an environment variable in the cluster rather than hard coding it here. We do need to distinguish dev/prod/etc. so we don't spam real users with email, etc.

Reply:
Yes, you are right. I used this just as a workaround to exiting build process: requires a dev or prod parameter. Actually, it doesn't matter much, we could use prod as well. Build process should be environment agnostic, I guess, to create a single docker image shared among all environments. Actual environment will be configured through the ConfigMap mounted in runtime. I'll take a look at build process to remove the need of a env parameter.

Reply:
This is still outstanding issue.